client/packages/common/src/ui/layout/tables/material-react-table/useTableFiltering.ts

```diff
@@ -37,32 +37,43 @@ export const useTableFiltering = <T extends MRT_RowData>(
   const filterUpdaters = useMemo(() => {
     const filterUpdaters: Record<string, (value: any) => void> = {};
 
-    columns.forEach(({ filterKey, id, accessorKey, filterVariant }) => {
```

Comment:
Review with "Ignore whitespace changes" enabled to see more easily the actual changes.